plugins/woocommerce/templates/emails/plain/email-fulfillment-details.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Email template for fulfillment details.
+ *
+ * @package WooCommerce\Templates\Emails
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+if ( null === $fulfillment->get_date_deleted() ) {
+	$tracking_number   = $fulfillment->get_meta( '_tracking_number', true );
+	$tracking_url      = $fulfillment->get_meta( '_tracking_url' );
+	$shipment_provider = $fulfillment->get_meta( '_shipment_provider' );
+	if ( ! $tracking_number && ! $tracking_url && ! $shipment_provider ) {
+		echo esc_html__( 'No tracking information available for this fulfillment at the moment.', 'woocommerce' );
+		return;
+	} else {
+		?>
+<strong>Tracking Number:</strong> <?php echo esc_attr( $tracking_number ); ?><br />
+<strong>Shipment Provider:</strong> <?php echo esc_html( $shipment_provider ); ?><br />
+<a href="<?php echo esc_html( $tracking_url ); ?>" target="_blank">Track your shipment</a><br /><br /><?php
```

Comment:
Since these are plain text emails, should we remove the html tags?

Reply:
Removed the html tags in plain emails in [Fix comments and optimize code](https://github.com/woocommerce/woocommerce/pull/58558/commits/0ca8834f8fe6bfb1018b617936eba6de7f9fc4cf).